openshift/setup-teardown/params/CUSTOM_SETTINGS/PR/api/tools/api-1-nodejs-bc.params

```diff
@@ -1,4 +1,5 @@
 NAME=pr-placeholder-eagle-api
+GROUP_NAME=pr-placeholder-epic
 SOURCE_REPOSITORY_URL=https://github.com/fork-placeholder/eagle-api
 SOURCE_REPOSITORY_REF=branch-placeholder
-NODE_ENV=development
+#NODE_ENV=development
```

Comment:
Should we just remove this?

Reply:
Yes I will remove that.  I don't really see a case where it's still needed